### ISSUE TITLE
a question about keras dropout in testing period

### ISSUE THREAD

**Issue by zhaojingyao** [spearsem/simpleRNN#2]
I have seen your comment 

There is another use case of dropout at testing or inference time: in order to get a notion of uncertainty and variability in the prediction of the network model, you might take a given input and run predict on it many times, each with different randomly assigned dropout neurons.

Say you run predict 100 times for a single test input. The average of these will approximate what you get with no dropout, the 'expected value' over different weight schemes. And various metrics like the standard deviation of these results will give you a sense of the error bounds of your estimate (conditioned on assumptions about the validity of the underlying model structure).

In this sense, it would be very useful to have to ability to re-activate Dropout settings from training, but specifically during testing or regular inference.

I want to know if I just predict a given input 100 times on a trained NN with dropout in training period and get a averaging model? Is there anything I should do in testing period?
